src/database/schema/ChatSchema.ts

```diff
@@ -280,6 +318,7 @@ class ChatSchema implements BaseDbSchema {
     userId: string,
     channelId: string,
     message: string,
+    attachments: any,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachments` parameter is not defined in the function parameters. It's better to define it explicitly.
```diff
-    attachments: any,
+    attachments: AttachmentType,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->